openapi.yaml

```diff
@@ -309,12 +309,12 @@ paths:
           application/json:
             schema:
               "$ref": "#/components/schemas/SchemaRequest"
-        required: true
+        required: false
```

Comment:
The `SchemaRequest` payload is not required? :o

Reply:
Oh, nice catch! That change is based on the swagger_spec that Murali updated. 
I think it should be safe to change that back to required, but I'll check with backend!

Reply:
It's false since a while now, I see. I used the openapi_spec for that, which was probably outdated. Would leave it like that for now to match the swagger doc!

Reply:
But this implies that you can send a schema request without a payload, which would do nothing? :o

Shouldn't we fix the mistakes in `openapi_spec` and/or imnprove accuracy when we write the documentation for those endpoints?

Reply:
I'll check with backend if it's a mistake!

Reply:
Updated! Can you take a look again? 🥺 🙏